fe1-web/ingestion/Handler.ts

```diff
@@ -6,7 +6,7 @@ import { Channel } from 'model/objects';
 import {
   addMessages, dispatch, OpenedLaoStore,
 } from 'store';
-import { handleMessage } from './handlers/Default';
+import { messageRegistry } from 'App';
```

Comment:
this will, unfortunately, create an import loop again, as App is the entrypoint and therefore imports (indirectly) the whole application.

we may be able to avoid the import here as well, by applying the good old principle of dependency injection :-)
Something along the lines of : 

```suggestion
let messageRegistry: MessageRegistry;

export function setMessageRegistry(r: MessageRegistry) {
  messageRegistry = r;
}
```

Then `ingestion/Configure.ts` can simply use the setter, no `import` involved 😄